src/main/java/com/zilen/weather/entity/Weather.java

```diff
@@ -0,0 +1,16 @@
+package com.zilen.weather.entity;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
Lombok is good thing for code reducing, but it would be better to write this code manually for the first time.